chalicelib/lib/slack.py

```diff
@@ -144,13 +147,24 @@ def slack_payload_extractor(req):
         return "failed extracting payload", 200
 
 
-def verify_token(token):
-    if token == os.getenv('slack_token'):
+def verify_token(request, token):
+    #https://api.slack.com/docs/verifying-requests-from-slack
```

Comment:
Remove this or add a real docstring with this information

Reply:
ok